test/e2e_node/runner/e2e/run_e2e.go

```diff
@@ -34,7 +34,7 @@ import (
 	"sync"
 	"time"
 
```

Comment:
What does the `e2e` in the path of this file imply? It's confusing because there's a lot of `e2e`s in that path. Maybe `jenkins` or `gce` would be a better directory name?